README.md

```diff
@@ -15,20 +15,56 @@
 |:----------------:|:----------------------------:|
 | v0.1.*           | v1.5.*                       |
 
-## Installation
+## Using this starter kit
 
-Add the following dependency to your Akeneo PIM project with this shell command:
+### Context
+To write an extension, you will need to integrate your extension in an existing PIM installation.
+We will use the PIM Community Standard Edition as the host of our connector.
 
+### Initialisation
+Create a new host project with the Standard Edition
+
+```
+composer create-project --prefer-dist akeneo/pim-community-standard /home/myhome/pim-project "1.5.*@stable"
+cd /home/myhome/pim-project
+```
+
+This will copy a copy of the standard edition without git informations.
+
+Then, clone the starter kit
+
+```
+composer require akeneo-labs/extension-starter 0.1.0 --prefer-dist
 ```
-composer require "akeneo-labs/extension-starter" "0.1.0";
+
+### Customization
+You will need to rename your extension according to the chosen name.
+Let's say you work for the WorldCompany and the connector name is WorldConnector.
+
+You will need to move your component inside the vendor directory according to your company name:
```

Comment:
You will have to move your component inside the vendor directory and rename it according to what vendor name you want to use. A widely used practice is to use your company name.